src/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,332 @@
+import React from 'react';
+import {
+    Bullseye,
+    Button,
+    ButtonVariant,
+    Card,
+    CardBody,
+    CardFooter,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Grid,
+    Label,
+    PageSection,
+    PageSectionVariants,
+    Stack,
+    StackItem,
+    Text,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ApacheKafkaLogo from './static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.svg';
+import APIMgmtLogo from './static/images/Logo-Red_Hat-OpenShift-API_Management-A-Standard-RGB.svg';
+import DataScienceLogo from './static/images/Logo-Red_Hat-OpenShift-Data_Science-A-Standard-RGB.svg';
+import ServiceRegistryLogo from './static/images/Logo-Red_Hat-OpenShift_Service_Registry-A-Standard-RGB.png';
+import DataServicesLogo from './static/images/Logo-Red_Hat-Data_Services-A-Standard-RGB.png';
+import './OverviewPage.css'
+
+import { useTranslation } from 'react-i18next';
+
+import { useHistory } from 'react-router-dom';
+
+export const OverviewPage: React.FunctionComponent = () => {
+    const { t } = useTranslation();
+
+    const history = useHistory();
+
+    const beta = location.pathname.startsWith('/beta');
+
+    const onClickKafkainstance = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        event.preventDefault();
+        history.push(`/streams/kafkas`);
+    };
+
+    const getKafkaHref = () => {
+        return history.createHref({
+            pathname: '/streams/kafkas',
+        });
+    };
+
+    const onClickServiceRegistry = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        if (beta) {
+            event.preventDefault();
+            history.push(`/service-registry`);
+        }
+    };
+
+    const getServiceRegistryHref = () => {
+        if (beta) {
+            history.createHref({
+                pathname: '/service-registry',
+            });
+        }
+        return '/beta/application-services/service-registry';
+    };
```

Comment:
All of these should be props passed to the component, how navigation works is an app-services-ui concern